docs/ecosystem-riot-testing-library.md

```diff
@@ -0,0 +1,36 @@
+---
+id: ecosystem-riot-testing-library
+title: riot-testing-library
+---
+
+[`riot-testing-library`][gh] builds on top of [DOM Testing Library](https://github.com/testing-library/dom-testing-library) by adding APIs for working with [Riot.js](https://riot.js.org/) components.
+
+```
+npm install --save-dev riot-testing-library
+```
+
+```javascript
+import render, { cleanup, fireEvent } from 'riot-testing-library'
+import TestTag from './test.tag'
+
+afterEach(() => {
+  cleanup()
+})
+
+test('should show count text  when rendered', () => {
+  const { queryByTestId } = render(TestTag, {count: 10});
+  expect(queryByTestId('count').textContent).toBe("10");
+})
+
+test('should add count when click add button text', async () => {
```

Comment:
Does this test need to be `async`?

Reply:
Thanks, this `async` is unnecessary,  I removed it